website/docs/reference/resource-configs/snowflake-configs.md

```diff
@@ -472,6 +472,8 @@ The [`incremental_strategy` config](/docs/build/incremental-strategy) controls h
 
```

Comment:
What if we move that last statement up and make this similar to the other ref pages

```suggestion

Snowflake supports the following incremental strategies:
- Merge (default)
- Append
- Delete+insert
- [`microbatch`](/docs/build/incremental-microbatch)

```